### PR TITLE
Safari UI fixes

### PR DESCRIPTION
**Pull request by jacogr**
- Minimum height for signer overlay, Fixes https://github.com/ethcore/parity/issues/3675
- onTouchTap for TabBar clicks, Fixes https://github.com/ethcore/parity/issues/3676

### CODE REVIEW

**Comment by ngotchac** [comment]
This breaks layout in Chrome (adds an unnecessary vertical scroll bar). Don't have Safari to test though

File: js/src/views/ParityBar/parityBar.css [hunk 1]
```diff
@@ -55,6 +55,7 @@
   overflow: auto;
   display: flex;
   background: rgba(0, 0, 0, 0.8);
+  min-height: 300px;
```
Comment (ngotchac) [inline js/src/views/ParityBar/parityBar.css]: This should be `min-height = .expanded.max-height (300) - (.header.height + 2*.header.padding)`, thus around `248px`

Comment (jacogr) [inline js/src/views/ParityBar/parityBar.css]: Think it is actually because we have 2 min-heights in there somewhere. (pending container as well)

Comment (jacogr) [inline js/src/views/ParityBar/parityBar.css]: Ok, got my macOS scrollbars to always show, one only on Chrome & Safari.

**Pull request merged**